python/python/bystro/examples/ExampleQueryToDataFrame.ipynb

```diff
@@ -0,0 +1,1306 @@
+       "      <td>.</td>\n",
+       "      <td>chr1:63200242:A:G</td>\n",
+       "      <td>[LINC00466]</td>\n",
+       "      <td>NaN</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>8</th>\n",
+       "      <td>chr1</td>\n",
+       "      <td>63318190</td>\n",
+       "      <td>63318190</td>\n",
+       "      <td>T</td>\n",
+       "      <td>C</td>\n",
+       "      <td>SNP</td>\n",
+       "      <td>.</td>\n",
+       "      <td>chr1:63318190:T:C</td>\n",
+       "      <td>[None]</td>\n",
+       "      <td>NaN</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>9</th>\n",
+       "      <td>chr1</td>\n",
+       "      <td>64236958</td>\n",
+       "      <td>64236958</td>\n",
+       "      <td>T</td>\n",
+       "      <td>C</td>\n",
+       "      <td>SNP</td>\n",
+       "      <td>.</td>\n",
+       "      <td>chr1:64236958:T:C</td>\n",
+       "      <td>[UBE2U]</td>\n",
+       "      <td>NaN</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  chrom       pos    vcfPos inputRef alt type           id              locus  \\\n",
+       "0  chr1  62416010  62416010        A   G  SNP   rs76667454  chr1:62416010:A:G   \n",
+       "1  chr1  62900720  62900720        A   C  SNP  rs139334931  chr1:62900720:A:C   \n",
+       "2  chr1  62978489  62978489        A   G  SNP            .  chr1:62978489:A:G   \n",
+       "3  chr1  63134348  63134348        A   G  SNP            .  chr1:63134348:A:G   \n",
+       "4  chr1  63146829  63146829        T   C  SNP            .  chr1:63146829:T:C   \n",
+       "5  chr1  63089877  63089877        T   A  SNP   rs72923353  chr1:63089877:T:A   \n",
+       "6  chr1  63178840  63178840        C   T  SNP            .  chr1:63178840:C:T   \n",
+       "7  chr1  63200242  63200242        A   G  SNP            .  chr1:63200242:A:G   \n",
+       "8  chr1  63318190  63318190        T   C  SNP            .  chr1:63318190:T:C   \n",
+       "9  chr1  64236958  64236958        T   C  SNP            .  chr1:64236958:T:C   \n",
+       "\n",
+       "  refSeq.name2  gnomad.exomes.AF  \n",
+       "0       [None]               NaN  \n",
+       "1       [None]               NaN  \n",
+       "2       [None]               NaN  \n",
+       "3       [None]               NaN  \n",
+       "4       [None]               NaN  \n",
+       "5       [None]               NaN  \n",
+       "6  [LINC00466]               NaN  \n",
+       "7  [LINC00466]               NaN  \n",
+       "8       [None]               NaN  \n",
+       "9      [UBE2U]               NaN  "
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# An example of a large query\n",
+    "import time\n",
+    "\n",
+    "start = time.time()\n",
+    "query_result_df_big_daly = get_annotation_result_from_query(\n",
```

Comment:
I know this is the largest dataset we can use right now, but I think would change the name or use a public dataset for example datasets, especially if people want to test the same dataset and get the same result, or just put down the stats for how long it takes for a dataset of size ___ .